<?xml version="1.0"?>
<!-- vim: set fenc=utf-8 et sts=4 ft=xml : -->
<!-- -*- mode: XML -*- -->

<launch>
	<node
		machine="search"
		name="beacon_finder"
		pkg="beacon_finder"
		type="beacon_finder.py"
		output="log">
	<param name="num_rows" type="double" value="4"/>
	<param name="num_columns" type="double" value="11"/>
	<param name="corner_circles_vertical_distance_meters" type="double" value="0.6985"/>
	<param name="camera_image_sensor_width_mmeters" type="double" value="15.6"/>
	<param name="camera_image_sensor_height_mmeters" type="double" value="23.6"/>
	<param name="blob_color" type="double" value="0"/>
	<param name="blob_min_area" type="double" value="9"/>
	<param name="blob_max_area" type="double" value="5000"/>
	<param name="blob_min_threshold" type="double" value="20"/>
	<param name="blob_max_threshold" type="double" value="220"/>
	<param name="blob_threshold_step" type="double" value="10"/>
	<param name="blob_min_distance_between_blobs" type="double" value="3.0"/>
	<param name="blob_repeatability" type="double" value="3L"/>
	<param name="do_histogram_equalization" type="bool" value="False"/>
	<param name="do_publish_debug" type="bool" value="True"/>
	</node>
</launch>
